### ISSUE TITLE
An issue about the selection of training set.

### ISSUE THREAD

**Issue by chenyulu2000** [davidnvq/visdial#38]
Sorry to bother you. Why is the **train+val set** used for evaluating the Visdial v1.0 validation set? Wouldn't this approach lead to an unusually high final accuracy?
![res](https://github.com/davidnvq/visdial/assets/61350652/ecc3ebb6-55b8-4a37-bf4d-654141416a63)
